### DISCUSSION TITLE
Getting error while searching for issues in comments; Requested value 'Mannequin' was not found.

### DISCUSSION THREAD

**Discussion by v-ankitgupta** [community/community#40924] [Enterprise]
### Select Topic Area

Question

### Body

The concerned repository is a private repo on GitHub Enterprise. Below code is throwing following error:

**Requested value 'Mannequin' was not found.**

`SearchIssuesRequest objCommentReq = new SearchIssuesRequest(label);`
`objCommentReq.In = new IssueInQualifier[] { IssueInQualifier.Comment };`
`objCommentReq.Repos.Add(currentProject.Url.Split('/')[3] + "/" + currentProject.Url.Split('/')[4]);`
`SearchIssuesResult resultFromComment = await gitHubClient.Search.SearchIssues(objCommentReq);`

Please advise. Thanks in advance.

**Comment by ossanna16** [comment]
Hi there @v-ankitgupta and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
